src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectChangeEventArgs.cs

```diff
@@ -25,14 +25,14 @@ public ProjectChangeEventArgs(IProjectSnapshot? older, IProjectSnapshot? newer,
         Kind = kind;
         SolutionIsClosing = solutionIsClosing;
         ProjectFilePath = older?.FilePath ?? newer?.FilePath;
-        ProjectKey = older?.Key ?? newer?.Key;
+        ProjectKey = (older ?? newer)!.Key;
```

Comment:
I'm surprised this causes a nullability warning. Looks like the flow analysis could be better. 😞

Reply:
Yeah this surprised me too, and also explains why I didn't realise when I originally added it, that the property didn't need to be nullable. Actually I should fix up `ProjectFilePath` while I'm here.